socket-server/scripts/getAllGoogleLanguagesSupport.go

```
@@ -0,0 +1,76 @@
+	}
+
+	fmt.Println("Using credentials:", credentials)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	client, err := translate.NewTranslationClient(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer client.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `client.Close` is not checked

(errcheck)